src/main/java/org/jboss/xavier/integrations/route/RouteBuilderExceptionHandler.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.xavier.integrations.route;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.builder.RouteBuilder;
+import org.jboss.xavier.integrations.jpa.service.AnalysisService;
+
+import javax.inject.Inject;
+import java.util.Map;
+
+import static org.jboss.xavier.integrations.route.MainRouteBuilder.ANALYSIS_ID;
```

Comment:
Could we refactor all the static fields from `MainRouteBuilder` to `RouteBuilderExceptionHandler` in order to avoid to have this class to import from one of the class that extends it?

Reply:
+1